#! /bin/sh -e 
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Description: Add driver_data member to vio_dev.
## DP: Patch author: Bastian Blank <waldi@debian.org>
## DP: Upstream status: unknown

diff -aurN a/include/asm-ppc64/vio.h b/include/asm-ppc64/vio.h
--- a/include/asm-ppc64/vio.h	2005-06-17 15:48:29.000000000 -0400
+++ b/include/asm-ppc64/vio.h	2005-06-18 12:02:58.000000000 -0400
@@ -91,6 +91,7 @@
 	char *type;
 	uint32_t unit_address;	
 	unsigned int irq;
+	void *driver_data;
 
 	struct device dev;
 };
